isa-parser/Makefile

```diff
@@ -14,17 +14,20 @@ export E Q
 PWD:=$(shell pwd)
 include $(PWD)/../config.mk
 
-install:
+install: deps
 	$(E) "Installing.."
-	$(CABAL) install $(CABALFLAGS)
+	$(CABAL) install $(INSTALL_FLAGS)
 
-pinstall: CABALFLAGS += --enable-executable-profiling
+pinstall: CONFIG_FLAGS += --enable-executable-profiling
 pinstall: setup-config
-	$(CABAL) install $(CABALFLAGS)
+	$(CABAL) install $(CONFIG_FLAGS)
+
+deps:
+	$(CABAL) install --only-dependencies
```

Comment:
There might be ways that I'm not aware of, but frankly I don't see the global option very useful in practice. For people who understand cabal, they normally won't do that anyway, and for people who don't, global installation just make them more opposed to risks.